sample-apps/donet-sample-app/Dockerfile

```diff
@@ -5,7 +5,9 @@ RUN dotnet publish *.csproj -c Release -o out
 
 FROM mcr.microsoft.com/dotnet/aspnet:7.0
 WORKDIR /app
-EXPOSE 8080
+COPY . ./
+ENV AWS_REGION=us-west-2
+ENV OTEL_EXPORTER_OTLP_ENDPOINT=http://otel:4317
 COPY --from=build-env /app/out .
-ENTRYPOINT ["dotnet", "dotnet-sample-app.dll"]
+ENTRYPOINT ["dotnet", "donet-sample-app.dll"]
```

Comment:
```suggestion
ENTRYPOINT ["dotnet", "dotnet-sample-app.dll"]
```

Reply:
This is not a typo, the project folder is called `donet-sample-app`.

Reply:
The folder should be renamed then

Reply:
I have actually renamed the project dotnet-sample-app to maintain consistency now.